hack/make/release-deb

```diff
@@ -49,41 +49,46 @@ if [[ ! "${components[*]}" =~ $component ]] ; then
 	components+=( $component )
 fi
 
-# create/update apt-ftparchive file
-if [ ! -f "$APTDIR/conf/apt-ftparchive.conf" ]; then
-	cat <<-EOF > "$APTDIR/conf/apt-ftparchive.conf"
-	Dir {
-		ArchiveDir "${APTDIR}";
-		CacheDir "${APTDIR}/db";
-	};
-
-	Default {
-		Packages::Compress ". gzip bzip2";
-		Sources::Compress ". gzip bzip2";
-		Contents::Compress ". gzip bzip2";
-	};
-
-	TreeDefault {
-		BinCacheDB "packages-\$(SECTION)-\$(ARCH).db";
-		Directory "pool/\$(SECTION)";
-		Packages "\$(DIST)/\$(SECTION)/binary-\$(ARCH)/Packages";
-		SrcDirectory "pool/\$(SECTION)";
-		Sources "\$(DIST)/\$(SECTION)/source/Sources";
-		Contents "\$(DIST)/\$(SECTION)/Contents-\$(ARCH)";
-		FileList "$APTDIR/\$(DIST)/\$(SECTION)/filelist";
-	};
-	EOF
+# If file exists, remove it since we will create a new one on every run.
+# This is essential inorder to not refresh bits (even the Releases file)
+# from EOL-ed distros.
+if [ -f "$APTDIR/conf/apt-ftparchive.conf" ]; then
+	rm "$APTDIR/conf/apt-ftparchive.conf"
+fi
```

Comment:
Why explicitly remove and not just delete the `if` letting `cat <<-EOF > "$APTDIR/conf/apt-ftparchive.conf"` just overwrite?